CHANGELOG.md

```diff
@@ -65,30 +76,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Fixed
 
-- [#1579](https://github.com/regen-network/regen-ledger/pull/1579) Return gRPC response code in query response.
+- [#1579](https://github.com/regen-network/regen-ledger/pull/1579) Return gRPC response code in query response
 
 ### x/ecocredit
 
-#### API Breaking Changes
-
-- [#1337](https://github.com/regen-network/regen-ledger/pull/1337) The `NewModule` method in `ecocredit/module` requires an `authority` address.
-- [#1337](https://github.com/regen-network/regen-ledger/pull/1337) The `NewKeeper` method in `ecocredit/core` requires an `authority` address.
-- [#1337](https://github.com/regen-network/regen-ledger/pull/1337) The `AddCreditType` proposal handler has been removed.
-- [#1342](https://github.com/regen-network/regen-ledger/pull/1342) The `NewKeeper` method in `ecocredit/marketplace` requires an `authority` address.
-- [#1342](https://github.com/regen-network/regen-ledger/pull/1342) The `AllowedDenom` proposal handler has been removed.
-- [#1357](https://github.com/regen-network/regen-ledger/pull/1357) The `NewModule` method in `ecocredit/module` requires a store `key`.
-- [#1462](https://github.com/regen-network/regen-ledger/pull/1462) Bridge operations must now specify a target/source that exists in the `AllowedBridgeChain` table.
-- [#1476](https://github.com/regen-network/regen-ledger/pull/1476) `DefaultParams` function has been removed.
-- [#1476](https://github.com/regen-network/regen-ledger/pull/1476) Remove `params` argument from `ValidateGenesis`.
-- [#1543](https://github.com/regen-network/regen-ledger/pull/1543) Remove `paramsKeeper` argument from `base/Keeper`, `basket/Keeper` and `marketplace/Keeper`.
```

Comment:
These items have been moved to their respective sections. I think we should be following the format we link to at the top of the document (at least for now) and this was misleading as it is incomplete.

Reply:
The migration guide is where we outline breaking changes for developers consuming the API. I also think we are the only developers importing the repository at the moment and using the golang API but breaking changes in the golang should be included in the `Changes` section and/or other sections where appropriate.

Reply:
We did a better job of using semantic commits this round but not quite ready to use for auto-generating a changelog, which I still think is what we should continue working towards, which would have the breaking change indicator (i.e. `!`).

Reply:
We did a better job of using semantic commits this round but not quite ready to use for auto-generating a changelog, which is what we should continue working towards and would have the breaking change indicator (i.e. `!`).